nav2_dwb_controller/dwb_controller/src/dwb_controller.cpp

```diff
@@ -34,7 +34,8 @@ DwbController::DwbController()
   RCLCPP_INFO(get_logger(), "Creating");
 
   // The costmap node is used in the implementation of the DWB controller
-  costmap_ros_ = std::make_shared<nav2_costmap_2d::Costmap2DROS>("local_costmap");
+  costmap_ros_ = std::make_shared<nav2_costmap_2d::Costmap2DROS>(
```

Comment:
The local namespace + name seems to be the same, is that something you expect to be true for the costmaps? Maybe we can reduce this to a single parameter?

Reply:
Yes, we could just pass one argument for both and keep the current namespacing:

```
Robot1/global_costmap/global_costmap
Robot1/local_costmap/local_costmap
```

However, I'm thinking that probably better naming could be:
```
Robot1/dwb/costmap
Robot2/world_model/costmap
```

In which case the local namespace and node's name are now different.

Reply:
However, this not a change I'm planning to do now.

Reply:
I think that might be a pain if you want to do something like having another server subbed to the costmap to run some calculation, since then the costmap will depend on the controller type. I just think its nicer to have the first version, but I don't have a strong opinion either way